src/cargo/core/features.rs

```diff
@@ -908,15 +914,19 @@ fn parse_git(it: impl Iterator<Item = impl AsRef<str>>) -> CargoResult<Option<Gi
 #[derive(Debug, Copy, Clone, Default, Deserialize)]
 pub struct GitoxideFeatures {
     /// All fetches are done with `gitoxide`, which includes git dependencies as well as the crates index.
+    #[serde(default = "default_true")]
```

Comment:
I assume these were chosen to align with `GitoxideFeatures::safe()` and to mirror if you do `-Zgitoxide`

However, this breaks down if you do `CARGO_UNSTABLE_GITOXIDE_FETCH=true`, then almost everything else becomes enabled (we should have a test to show this case).

Some other challenges with these defaults
- Unclear what the motivation is (could have comment pointing to `GitoxideFeature::safe`)
- Challenge in keeping in sync with `GitoxideFeature::safe`

Should we instead just default everything to false?

Reply:
What I would love to just work is `CARGO_UNSTABLE_GITOXIDE=fetch,list_files` mirroring the CLI flag (which would also imply you could have `unstable.gitoxide = "fetch,list_files"` in the config file).

Reply:
Yeah maybe we can make it a comma separated list to reduce the complexity. 

Wonder do people really use individual gitoxide features? I always assume that people would just enable them as a whole, though gitoxide author recommends keeping them separately: <https://github.com/rust-lang/cargo/pull/13252#discussion_r1447868466>.

Reply:
> What I would love to just work is CARGO_UNSTABLE_GITOXIDE=fetch,list_files and CARGO_UNSTABLE_GITOXIDE=true mirroring the CLI flag (which would also imply you could have unstable.gitoxide = "fetch,list_files" in the config file).

So what we defined as table is going to be replaced with a string? It makes sense,  I have no objection to this, but it is not clear that it will break the configuration of the previous version？

Reply:
It's possible to support both, via `#[serde(deserialize_with = ...)]` and a custom deserializer that handles a string/list or a struct, like 

https://github.com/rust-lang/cargo/blob/88d55a89e0cb596a4bda53ce194b6753c08a3798/src/cargo/util/context/mod.rs#L2673